.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,5 @@
 blank_issues_enabled: true
 contact_links:
-  - name: Ask Question
-    url: https://github.com/scaffold-eth/scaffold-eth-2/discussions/new?category=q-a
-    about: Ask questions and discuss with other community members
   - name: Request Feature
-    url: https://github.com/scaffold-eth/scaffold-eth-2/discussions/new?category=ideas 
-    about: Requests features or brainstorm ideas for new functionality
\ No newline at end of file
+    url: https://github.com/BuidlGuidl/abi.ninja/discussions/new?category=ideas
```

Comment:
Didn't remove this since I feel its nice idea to put genreal feature request to discussion's this doesn't clutter issues tab but we can remove this as well 

If we plan to keep it maybe @carletex can open up https://github.com/BuidlGuidl/abi.ninja/discussions with ideas

Reply:
Opened the discussions!